DocParser/JSDocHelper.cs

```diff
@@ -0,0 +1,255 @@
+                    {
+                        results.List.Add(new KeyValuePair<string, string>(key, sb.ToString().Trim()));
+                    }
+
+                    sb.Clear();
+
+                    var iAt = line.IndexOf(' ');
+                    if (iAt > -1)
+                    {
+                        key = line.Substring(1, iAt - 1);
+                        sb.Append(line.Substring(iAt + 1) + ' ');
+                    }
+                    else
+                    {
+                        // issue: if an @indicator is alone on a line, followed by the actual text, this will miss it.
+                        key = line.Substring(1).Trim();
+                        results.List.Add(new KeyValuePair<string, string>(key, sb.ToString().Trim()));
+                        key = string.Empty;
+                    }
+                }
+                else
+                {
+                    sb.Append(line + ' ');
+                }
+            }
+
+            if (!string.IsNullOrWhiteSpace(key) || sb.Length > 0)
+            {
+                results.List.Add(new KeyValuePair<string, string>(key, sb.ToString().Trim()));
+            }
+
+            results.PreviousLine = set.PreviousLine;
+            results.NextLine = set.NextLine;
+
+            return results;
+        }
+
+        // Assumes text contains /** ... */ commenting. 'text' to contain a single comment block.
+        // Returns lines without comments
+        public static List<string> GetLines(string text)
+        {
+            var results = new List<string>();
+
+            var lines = text.Split(new[] { '\n' });
+            foreach (var fullline in lines)
+            {
+                var line = fullline.Trim();
+                var iAt = line.IndexOf('*');
+                if (iAt > -1)
+                {
+                    line = line.Substring(iAt + 1).Trim();
+                }
+
+                if (string.IsNullOrWhiteSpace(line))
+                {
+                    continue;
+                }
+
+                results.Add(line);
+            }
+
+            return results;
+        }
+
+        public static string[] SplitFrom(string text, string from, string between)
+        {
+            var iAt = text.IndexOf(from);
+            if (iAt>-1)
+            {
+                text = text.Substring(iAt + from.Length);
+            }
+
+            return text.Split(new string[] { between }, StringSplitOptions.RemoveEmptyEntries);
+        }
+
+        public static string[] Split(string text, string left, string right)
+        {
+            var list = new List<string>();
+            var iAt = -1;
+            iAt = text.IndexOf(left, iAt+1);
+            while (iAt>-1)
+            {
+                var iAt2 = text.IndexOf(right, iAt+left.Length);
+                if (iAt2>-1)
+                {
+                    list.Add(text.Substring(iAt+left.Length, iAt2 - iAt - left.Length));
```

Comment:
inconsistent spacing between expressions, should have spaces around operators.  (not just here)